docs/synapse/userguides/analytical_model_tags.ipynb

```diff
@@ -236,7 +236,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "hideCode": false
+    "hideCode": true
```

Comment:
I explicitly left this as False so that a use can see how this is done with a Cortex proxy.

Reply:
@therealsilence Are you intending to hide this user facing example?